Samples.xcodeproj/project.pbxproj

```diff
@@ -59,12 +59,14 @@
 		00E5401C27F3CCA200CF66D5 /* SamplesApp.swift in Sources */ = {isa = PBXBuildFile; fileRef = 00E5400C27F3CCA100CF66D5 /* SamplesApp.swift */; };
 		00E5401E27F3CCA200CF66D5 /* ContentView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 00E5400D27F3CCA100CF66D5 /* ContentView.swift */; };
 		00E5402027F3CCA200CF66D5 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 00E5400E27F3CCA200CF66D5 /* Assets.xcassets */; };
-		1C0C1C3929D34DAE005C8B24 /* ChangeViewpointView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1C0C1C3429D34DAE005C8B24 /* ChangeViewpointView.swift */; };
```

Comment:
Can you help fix the project file by select "Sort by Name" on the Samples folder, in the Project Navigator (the Xcode file pane on the left)? Xcode will apply some autofixes. It seems another PR made [some unintended changes](https://github.com/Esri/arcgis-maps-sdk-swift-samples/pull/146/files#diff-edb149324ca147e6068a5e155551b365ff9044bd6b606d0a6803dbb928e696fcR185) that slipped through my eyes. Thank you.

Reply:
`Sort by name` will remove the following files from the project file, is this expected?
```
08EC04329D25D6D000F35D0 /* ChangeMapViewBackgroundView.swift in Copy Source Code Files */,
108EC04429D25D6D000F35D0 /* ChangeMapViewBackgroundView.SettingsView.swift in Copy Source Code Files */,
108EC04529D25D6D000F35D0 /* ChangeMapViewBackgroundView.Model.swift in Copy Source Code Files */,
```

Reply:
Yes, I think these entries are duplicates. To verify, you can go to the copy readme file build phase and see if these files are still in that list.